<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Manish-PC" ignored="0" name="Test" tests="4" failures="4" timestamp="16 Mar 2019 03:16:12 GMT" time="0.108" errors="0">
  <testcase name="AddtoCart" time="0.033" classname="tests.FunctionalTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pageObjects.Products.Search_Bar(Products.java:26)
at tests.FunctionalTest.AddtoCart(FunctionalTest.java:52)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- AddtoCart -->
  <testcase name="Home" time="0.004" classname="tests.FunctionalTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pageObjects.HomePage.SignOut(HomePage.java:21)
at tests.FunctionalTest.Home(FunctionalTest.java:17)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Home -->
  <testcase name="Login" time="0.003" classname="tests.FunctionalTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pageObjects.LoginPage.Login(LoginPage.java:23)
at tests.FunctionalTest.Login(FunctionalTest.java:25)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase name="LoginFail" time="0.002" classname="tests.FunctionalTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pageObjects.LoginPage.Login(LoginPage.java:23)
at tests.FunctionalTest.LoginFail(FunctionalTest.java:37)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- LoginFail -->
</testsuite> <!-- Test -->
